### DISCUSSION TITLE
File matching pattern example is misleading

### DISCUSSION THREAD

**Discussion by jlai403** [community/community#48362] [Actions]
### Select Topic Area

General

### Body

the `**.js` pattern match example documented [here](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#patterns-to-match-file-paths<title>) doesn't return `.js` files in subdirectories, only to the root. I had to use the `**/**.js` pattern for it to return all matches in my repo. 

The match examples in the documentation for this is mis-leading.

**Comment by jsoref** [comment]
Please use https://github.com/github/docs/issues for issues w/ the docs.

Once you've filed an issue, please reply to this discussion with the link to the issue (you can mark that as the answer)

**Comment by jlai403** [comment]
https://github.com/github/docs/issues/24132
